### PR TITLE
Repair of broken links - 2nd iteration

### PR DESCRIPTION
**Pull request by aninde**
### Context

It seems that despite the adjustments at https://github.com/handsontable/handsontable/pull/8192 there are still some links to fix. This PR solves this issue by adding `@` where it is missing.
### How has this been tested?

Manually after `npm run docs:build`.

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1. https://github.com/handsontable/handsontable/issues/8306
2.
3.

### Affected project(s):
- [ ] `handsontable`
- [ ] `@handsontable/angular`
- [ ] `@handsontable/react`
- [ ] `@handsontable/vue`
- [x] `documentation`

### Checklist:

- [x] I have reviewed the guidelines about [Contributing to Handsontable](https://github.com/handsontable/handsontable/blob/master/CONTRIBUTING.md) and I confirm that my code follows the code style of this project.
- [ ] I have signed the [Contributor License Agreement](https://docs.google.com/forms/d/e/1FAIpQLScpMq4swMelvw3-onxC8Jl29m0fVp5hpf7d1yQVklqVjGjWGA/viewform?c=0&w=1)
- [ ] My change requires a change to the documentation.

### CODE REVIEW

File: docs/next/guides/building-and-testing/plugins.md [hunk 1]
```diff
@@ -252,7 +252,7 @@ const hotInstance = new Handsontable(container, {
 ```
 
 ### 5. Get a reference to the plugin's instance
-To use the plugin's API, call the [`getPlugin`](../api/core/#getplugin) method to get a reference to the plugin's instance.
+To use the plugin's API, call the [`getPlugin`](@/../../../api/core/#getplugin) method to get a reference to the plugin's instance.
```
[the remark this answers is not in the document]
Comment (aninde) [reply docs/next/guides/building-and-testing/plugins.md]: Done by a0d9827de880b9f511350963bf56c4a88a6d255a.

**Review by budnix — APPROVED** [review]
LGTM 👍

File: docs/next/api/introduction.md [hunk 2]
```diff
@@ -11,7 +11,7 @@ canonicalUrl: /api/
 
 Welcome to Handsontable API reference. Our goal is make it easy to dive in and start coding from day one.
 
-However, data grids are rather complex libraries, so we assume that you possess a certain level of expertise in JavaScript before moving forward. If you want to grab some basics, we have provided a JavaScript example in the [Hello World](../hello-world/) section.
+However, data grids are rather complex libraries, so we assume that you possess a certain level of expertise in JavaScript before moving forward. If you want to grab some basics, we have provided a JavaScript example in the [Hello World](@/hello-world/) section.
```
Comment (Siemienik) [inline docs/next/api/introduction.md:14]: I suppose that is incorrect. Could you check this link once again?

**Review by Siemienik — CHANGES_REQUESTED** [review]

File: docs/next/api/introduction.md (hunk 2 above)
Comment (Siemienik) [reply docs/next/api/introduction.md:14]: Afaik: the proper link is `[Hello World](@/guides/hello-world.md)`

Comment (aninde) [reply docs/next/api/introduction.md]: Thank you. The correct link that I tested is in e23c4afd59a15eebee5c66ae443bed06fdd51a6a.

**Review by Siemienik — APPROVED** [review]
LGTM 👍

**Pull request merged**